### DISCUSSION TITLE
New SD.Next logo for browser tab and desktop shortcuts

### DISCUSSION THREAD

**Discussion by Centurion-Rome** [vladmandic/sdnext#1657] [Ideas]
It would be nice to have some minimalistic  SD.Next logo for browser tab and desktop shortcuts.
My last refinement creation from Discord. original idea and kudos to @therug + @TanukiKanawa, + Some Feedback Vlad, Raivshard
 I am still refining it 🙂
![sSD-Next-15](https://github.com/vladmandic/automatic/assets/75789136/aee2d6aa-6c2f-46db-b22d-3400a698776a)

**Comment by vladmandic** [comment]
there is quite a long discussion thread in discord on that topic, join there?

**Reply by Centurion-Rome** [reply]
Ok, I will follow your suggestion :)

**Reply by Centurion-Rome** [reply]
posted to "Emojis ideas"

**Reply by vladmandic** [reply]
Nope, that's for emojis to be added to discord.
There is actual logo channel, it's just not top level.

**Reply by Centurion-Rome** [reply]
Found it was in Feedback=>New Logo Ideas.
=> posted

Thanks for the tip and have a great day :)
